src/QbXml/Objects/Types/DATETIMETYPE.cs

```diff
@@ -167,7 +167,9 @@ public void ReadXml(System.Xml.XmlReader reader)
             reader.ReadStartElement();
             if (!isEmptyElement)
             {
-                value = DateTime.Parse(reader.ReadContentAsString());
+                DateTime dt;
+                DateTime.TryParse(reader.ReadContentAsString(), out dt);
+                value = dt;
```

Comment:
It should all still work as expected. Calling `ToString()` will still give you the string value--but it would be whatever `DateTime.MinValue` is if it couldn't parse the date/time.